init-tools.sh

```diff
@@ -22,47 +22,59 @@ case $OSName in
 
     Linux)
         OS=Linux
-        __DOTNET_PKG=dotnet-ubuntu-x64
+        source /etc/os-release
+        if [ $ID = "centos" ]; then
```

Comment:
Shouldn't this require '==' instead of just '='? Same for all other comparisons.